arch/x86/kvm/vmx/pkvm/hyp/io_emulate.c

```diff
@@ -108,7 +108,7 @@ static int emulate_host_pio(struct kvm_vcpu *vcpu, struct pkvm_pio_req *req)
 	return ret;
 }
 
-bool pio_access_valid(int size)
+static bool pio_access_valid(int size)
```

Comment:
`error: static declaration of 'pio_access_valid' follows non-static declaration`

So it's better to move `pio_access_valid()` before `get_pio_handler()`.

Besides that, look like `pio_access_valid()` should be made static already in https://github.com/intel-staging/pKVM-IA/pull/22/commits/0a0b5bb10c183308988ab0303c8821b262a7f192 where it is introduced,  not in this patch which is about MMIO.

Reply:
Sorry my fault. Fixed it.